src/lib/CLValue/PublicKey.ts

```diff
@@ -165,7 +176,7 @@ export class CLPublicKey extends CLValue {
 
   /**
    * Tries to decode PublicKey from its hex-representation.
-   * The hex format should be as produced by PublicKey.toAccountHex
+   * The hex format should be as produced by CLPublicKey.toHex
    * @param publicKeyHex
    */
   static fromHex(publicKeyHex: string): CLPublicKey {
```

Comment:
maybe an optional second parameter here to force checksumed variant of pubkey? then it can throw an error that the value might be invalid because we can't validate it? and for dapp develop it will be handy to just `try catch` and handle in his app.